.github/workflows/static-build.yml

```
@@ -1,16 +1,20 @@
+# ==============================================================================
+# Protocol    : Deep State of Mind (DSOM) For My AI
+# Author      : Harisfazillah Jamel (LinuxMalaysia)
+# Timestamp   : 2026-07-26
+# License     : GNU General Public License v3.0
+# Standard    : UK English | DBP-standard Bahasa Melayu Malaysia (Piawai)
+# ==============================================================================
 name: Bake PHP to Static HTML
 on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)